xray/lambda.go

```diff
@@ -68,21 +69,19 @@ func createFile(dir string, name string) (string, error) {
 	filePath := fileDir + string(os.PathSeparator) + name
 
 	// detect if file exists
-	var _, err = os.Stat(filePath)
+	_, err := os.Stat(filePath)
 
 	// create file if not exists
 	if os.IsNotExist(err) {
-		e := os.MkdirAll(dir, os.ModePerm)
-		if e != nil {
-			return filePath, e
-		} else {
-			var file, err = os.Create(filePath)
-			if err != nil {
-				return filePath, err
-			}
-			file.Close()
-			return filePath, nil
+		if e1 := os.MkdirAll(dir, os.ModePerm); e1 != nil {
```

Comment:
Could you provide unit tests for code refactoring about this part?

Reply:
I'll see what I can do, but I suspect it is already tested. this minor refactor doesn't change the logic.

Reply:
Have you test this part already? I think it's more safe if we test the code before we merge. We cannot 100% sure that our change won't break anything even if it looks like there is no logic error.

Reply:
The only reason I changed it was to comply with golint and govet. I reverted to the original code and disabled the linters.